Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/FileTransporter.cs

```diff
@@ -95,6 +95,7 @@ public FileTransferResult Run()
 
                 client.ConnectionInfo.Timeout = TimeSpan.FromSeconds(_batchContext.Connection.ConnectionTimeout);
                 client.KeepAliveInterval = TimeSpan.FromMilliseconds(_batchContext.Connection.KeepAliveInterval);
+                client.OperationTimeout = TimeSpan.FromSeconds(_batchContext.Connection.ConnectionTimeout);
 
                 client.BufferSize = _batchContext.Connection.BufferSize * 1024;
 

```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/16d43b54383d92fb4485b9f216199e8e2f34bdba/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/FileTransporter.cs#L433
If there is no use for this line, then please remove it.